src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php

```diff
@@ -512,6 +564,62 @@ public function get_item_schema() {
 					'context'     => array( 'embed', 'view', 'edit' ),
 					'readonly'    => true,
 				),
+				'variations'       => array(
+					'description' => __( 'Block variations.' ),
+					'type'        => 'array',
+					'items'       => array(
+						'type'       => 'object',
+						'properties' => array(
+							'name'        => array(
+								'description' => __( 'The unique and machine-readable name.' ),
+								'type'        => 'string',
+								'required'    => true,
+							),
+							'title'       => array(
+								'description' => __( 'A human-readable variation title.' ),
+								'type'        => 'string',
+								'required'    => true,
+							),
+							'description' => array(
+								'description' => __( 'A detailed variation description.' ),
+								'type'        => 'string',
+								'required'    => false,
+							),
+							'category'    => array(
```

Comment:
`category` can be shared with the block as well.

Reply:
Good eye, ✅ done in https://github.com/WordPress/wordpress-develop/pull/1015/commits/4fb70c610525ec7fee4d985a210d361a32ddf1b6